src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const populationElements = document.querySelectorAll('.population');
+
+const arr = [];
+
+populationElements.forEach((el) => {
+  arr.push(Number(el.innerHTML.split(',').join('')));
+});
+
+const total = arr.reduce((acc, i) => acc + i, 0);
+const avarage = total / arr.length;
```

Comment:
There is a typo in the variable name 'avarage'. It should be 'average' to match the correct spelling.